sbndqm/Decode/PMT/PMTDecodeData/classes_def.xml

```diff
@@ -1,6 +1,6 @@
 <lcgdict>
   
-  <class name="pmtAnalysis::PMTDigitizerInfo" />
+  <class name="pmtAnalysis::PMTDigitizerInfo" classVersion="10" />
```

Comment:
This looks suspicious: there should be a checksum here.
Could you try build again and see if GenReflex has added the checksum to this file? in that case, the new file should be committed to this pull request too.